backend/src/order.ts

```diff
@@ -13,6 +13,7 @@ import auth = require('./confluenceAuth');
 
 // Paytrail wants to charge something, so they don't support minimal payments
 const PAYTRAIL_MIN_PAYMENT = 0.65;
+const PAYTRAIL_PREFIX = 'LIPPUKAUPPA';
```

Comment:
Me too, but I tried it and paytrail didn't like it (it was shown in Paytrail as LIPPUKAUPPA?1), so I thought that maybe it's better not to have it.